backend/crud-server/src/routes/api/conduits.js

```diff
@@ -192,9 +194,11 @@ router.put('/:id', auth.required, async (req, res, next) => {
       return next(new RestApiError(422, errors));
     }
 
-    if (serviceTargets.includes(conduit.suriType) === false) {
+    if (serviceTargets.includes(req.body.conduit.suriType) === false) {
```

Comment:
Would be better to use spreading here eg:  const { conduit } = request.body; ?

Reply:
@vakareeralil  There is already a variable `conduit` in the same scope. we'd then need to alias this spread variable. Anyway these code will be refactored as part of next epic.